### PR TITLE
Small fixes for firecracker networking

### PR DESCRIPTION
**Pull request by bduffany**
* Add iptables rule to fix broken `veth1xxxxx <=> defaultDevice` link in the case where packet forwarding is disabled in the root netns. The rule is added per-container and is removed as part of container networking cleanup.
* Avoid adding multiple masquerade rules on each run of the executor

---

**Version bump**: Patch

### CODE REVIEW

**Review by tylerwilliams — APPROVED** [review]
LGTM

**Comment by bduffany** [comment]
Merging https://github.com/buildbuddy-io/buildbuddy/pull/1434/files as part of this PR, because I accidentally merged that one after it got LGTM but before this PR was merged.

**Pull request merged**
